### PR TITLE
[DE] Add `Discord servers`, `osu! University` and `osu! Medal Hunters`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/Discord_servers/de.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Discord-Server
+
+Indexseite für alle Artikel über [Discord-Server](https://discord.com/) aus der Community.
```
Comment (IceDynamix) [inline wiki/Community/Discord_servers/de.md:3]:
```suggestion
Indexseite für alle Artikel über [Discord-Server](https://discord.com/) der Community.
```
i think it feels more natural this way

File: wiki/Community/Discord_servers/osu!_Medal_Hunters/de.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+|  |  |
+| :-- | :-- |
+| Einladungslink | <https://discord.gg/osumedalhunters> |
+| Alternativer Einladungslink | <https://discord.gg/8qpNTs6> |
+
+:::
+
+*Hinweis: Die Zusammenarbeit bei der Suche nach Hush-Hush-Medaillen ist zwar erwünscht, aber es ist auch möglich, sie allein zu finden. Beachte, dass der Server Lösungen für Hush-Hush-Medaillen enthält.*
+
+**osu! Medal Hunters** ist der primär von der Medaillenjäger-Community genutzte [Discord-Server](https://discord.com), in dem Leute über  [Medaillen](/wiki/Medals) und [ihre Lösungen](/wiki/Medals/Unlock_requirements) diskutieren.
```
Comment (IceDynamix) [inline wiki/Community/Discord_servers/osu!_Medal_Hunters/de.md:27]:
```suggestion
**osu! Medal Hunters** ist der primär von der Medaillenjäger-Community genutzte [Discord-Server](https://discord.com), in dem Leute über [Medaillen](/wiki/Medals) und [ihre Lösungen](/wiki/Medals/Unlock_requirements) diskutieren.
```

File: wiki/Community/Discord_servers/osu!_University/de.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+| Twitch-Kanal | [osuuniversity](https://twitch.tv/osuuniversity) |
+| Twitter-Kürzel | [@osuuniversity](https://twitter.com/osuuniversity) |
+| Spotify-Kanal | [osu! University](https://open.spotify.com/show/1KJTvGLOmUTnvbZoZGLlMP) |
+| YouTube-Kanal | [osu! University](https://www.youtube.com/c/osuUniversity) |
+
+:::
+
+**osu! University** ist ein [Discord-Server](https://discord.com) mit Schwerpunkt auf der Diskussion über Verbesserungen, hauptsächlich im [osu!-Spielmodus](/wiki/Game_mode/osu!). Es werden regelmäßig verschiedene Events wie Interviews mit Spitzenspielern und [Turniere](/wiki/Tournaments) veranstaltet sowie Studien zur Optimierung durchgeführt und der Server ist auch bekannt dafür, kostenpflichtige Coaching-Dienste anzubieten.
+
+Der Server beherbergt auch den Platz für die Entwicklung, das Testen und den Support des Programms [Rewind](https://osu.ppy.sh/community/forums/topics/1451845), das ein Tool zur Analyse von [Replays](/wiki/Gameplay/Replay) ist.
```
Comment (IceDynamix) [inline wiki/Community/Discord_servers/osu!_University/de.md:23]:
```suggestion
Der Server beherbergt auch den Platz für die Entwicklung, das Testen und den Support der Anwendung [Rewind](https://osu.ppy.sh/community/forums/topics/1451845), ein Tool zur Analyse von [Replays](/wiki/Gameplay/Replay).
```

File: wiki/Community/Discord_servers/osu!_University/de.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+| :-- | :-- |
+| Einladungslink | <https://discord.gg/QubdHdnBVg> |
+| Twitch-Kanal | [osuuniversity](https://twitch.tv/osuuniversity) |
+| Twitter-Kürzel | [@osuuniversity](https://twitter.com/osuuniversity) |
+| Spotify-Kanal | [osu! University](https://open.spotify.com/show/1KJTvGLOmUTnvbZoZGLlMP) |
+| YouTube-Kanal | [osu! University](https://www.youtube.com/c/osuUniversity) |
+
+:::
+
+**osu! University** ist ein [Discord-Server](https://discord.com) mit Schwerpunkt auf der Diskussion über Verbesserungen, hauptsächlich im [osu!-Spielmodus](/wiki/Game_mode/osu!). Es werden regelmäßig verschiedene Events wie Interviews mit Spitzenspielern und [Turniere](/wiki/Tournaments) veranstaltet sowie Studien zur Optimierung durchgeführt und der Server ist auch bekannt dafür, kostenpflichtige Coaching-Dienste anzubieten.
```
Comment (IceDynamix) [inline wiki/Community/Discord_servers/osu!_University/de.md:21]:
```suggestion
**osu! University** ist ein [Discord-Server](https://discord.com) mit Schwerpunkt auf Diskussion über Verbesserung, hauptsächlich im [osu!-Spielmodus](/wiki/Game_mode/osu!). Es werden regelmäßig verschiedene Events wie Interviews mit Spitzenspielern und [Turniere](/wiki/Tournaments) veranstaltet sowie Studien zur Optimierung durchgeführt und der Server ist auch bekannt dafür, kostenpflichtige Coaching-Dienste anzubieten.
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**